study-builder/studies/mbc/snippets/common/composite-question.conf

```diff
@@ -0,0 +1,15 @@
+{
+  "questionType": "COMPOSITE",
```

Comment:
Why do we need this file? There's no translations here...

Reply:
You are right. Has to do with the method I followed: composite-question got "translated" by extracting all the English strings, modifying the file with a reference to the string and adding new entry for spanish. In this case, though, the number of English strings was 0.
If you don't mind, just let it be. There is a very little duplication and study builder seems to be doing right thing with how it is configured.